Add high agency as a value of the company (#1603)
sourcegraph/about: company/values.md

> I am working on improving the security of our product, but I feel uncomfortable clearly documenting what I am doing (e.g., in public status updates, issues, PR descriptions, the changelog) because I am worried that it will make Sourcegraph look bad.

Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.

> ## High Agency

Reviewer: ```suggestion
## High agency
```

sentence case, not title case (https://about.sourcegraph.com/handbook/communication/content_guidelines/actionable_language#headings-and-subheadings)

> Every person at Sourcegraph has both the power and the responsibility to help make Sourcegraph a better place and a better product.

Reviewer: ```suggestion
Everyone at Sourcegraph has the power and the responsibility to improve Sourcegraph as a company and as a product.
```

remove extraneous words, use more meaningful verbs

> I am working on improving the security of our product, but I feel uncomfortable clearly documenting what I am doing (e.g., in public status updates, issues, PR descriptions, the changelog) because I am worried that it will make Sourcegraph look bad.

Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.

## High Agency

Every person at Sourcegraph has both the power and the responsibility to help make Sourcegraph a better place and a better product.

> “Life can be much broader once you discover one simple fact: Everything around you that you call life was made up by people that were no smarter than you and you can change it, you can influence it, you can build your own things that other people can use. Once you learn that, you'll never be the same again.” - Steve Jobs

> - As a [handbook-first](https://about.sourcegraph.com/handbook/usage#why-handbook-first) company, all of us are free to discuss and implement changes to any aspect of our business.

Reviewer: This bullet list feels arbitrary. Why is it in bullets and the rest isn't?

Author: Agreed - was considering having a longer list of examples, but in current state, bullets don't make sense.

> - Engineers feel empowered to do what they feel needs to be done through our policy of [innovation time](https://about.sourcegraph.com/handbook/engineering#innovation-time).

Reviewer: This is a great example, but its inclusion implies (by omission) that non-engineers don't have this power. You could fix this by somehow indicating that this is just an example.

Author: Reworded along with removing bullets.

- [TODO: Talk about how other teams are encouraged to exercise their agency].

> This power of agency also comes with the responsibility of integrity and initiative. If you feel that something is important, then advocate for it! If you feel like you're using the wrong tools for the job or doing the wrong thing, then you're responsible for quickly voicing that concern, and engaging with your teammates to find the solution that is the best for the team.

Reviewer: To me, "integrity" means honesty and being moral. These things are obviously important, but I don't see how they apply here. This paragraph doesn't elaborate on integrity. Remove it?

Author: Agreed, this can be condensed into the final paragraph.

> This doesn't mean that everything must constantly be in flux. Rather, it means that, when you believe something should happen, you either make it happen, or find a better solution.

Reviewer: Are you concerned people will read the above and think "oh, wow, that must mean everything is in flux"? I personally don't have that concern. This paragraph IMO is not needed.

Author: Fair, this is too wordy and not super needed. Will remove.

> This also doesn't mean that you should make unilateral decisions. As an [open company](index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. It is not a bad outcome if your proposal is not accepted or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one!

Reviewer: Can you incorporate this point into the sentence above and make it something like "With agency comes responsibility: ... (bullet list with 1 bullet for each of integrity (if needed), initiative, and collaboration)".

> I am working on improving the security of our product, but I feel uncomfortable clearly documenting what I am doing (e.g., in public status updates, issues, PR descriptions, the changelog) because I am worried that it will make Sourcegraph look bad.

Our code and PRs are public, so lack of transparency is only obfuscation, not security. Unless something is immediately exploitable and would put customer data at risk, you should be completely transparent about it, just like any other non-security improvement.

## High Agency

> Every person at Sourcegraph has both the power and the responsibility to help make Sourcegraph a better place.

Reviewer 2: ```suggestion
Every person at Sourcegraph has both the power and the responsibility to help make Sourcegraph a better place and a better product.
```

> > “Life can be much broader once you discover one simple fact: Everything around you that you call life was made up by people that were no smarter than you and you can change it, you can influence it, you can build your own things that other people can use. And you can change it, you can influence it… Once you learn that, you'll never be the same again.” - Steve Jobs

Reviewer 3: It seems like there is some duplication in this quote that I assume is unintentional

```suggestion
> “Life can be much broader once you discover one simple fact: Everything around you that you call life was made up by people that were no smarter than you and you can change it, you can influence it, you can build your own things that other people can use. Once you learn that, you'll never be the same again.” - Steve Jobs
```

Author: Yep, good catch, thanks!

- As a [handbook-first](https://about.sourcegraph.com/handbook/usage#why-handbook-first) company, all of us are free to discuss and implement changes to any aspect of our business.
- Engineers feel empowered to do what they feel needs to be done through our policy of [innovation time](https://about.sourcegraph.com/handbook/engineering#innovation-time).
- [TODO: Talk about how other teams are encouraged to exercise their agency].

> This power of agency also comes with the responsibility of integrity. If you feel that something is important, then fight for it! We want a workplace where nobody feels like they're using the wrong tools for the job or doing the wrong thing. If they do, they're responsible for quickly voicing that concern, then proposing and executing upon solutions. Their teammates, then, are responsible for engaging in discussion around the topic, and working together to make the best choice for everyone.

Reviewer 3: (I think it is more powerful if it is consistently "you" instead of "they/their")

```suggestion
This power of agency also comes with the responsibility of integrity. If you feel that something is important, then advocate for it! If you feel like we're using the wrong tools for the job or doing the wrong thing, then you're responsible for quickly voicing that concern, and engaging with your teammates to find the solution that is the best for the team.
```

Author: That definitely flows better, thanks!

> This power of agency also comes with the responsibility of integrity. If you feel that something is important, then fight for it! We want a workplace where nobody feels like they're using the wrong tools for the job or doing the wrong thing. If they do, they're responsible for quickly voicing that concern, then proposing and executing upon solutions. Their teammates, then, are responsible for engaging in discussion around the topic, and working together to make the best choice for everyone.

Reviewer 2: ```suggestion
This power of agency also comes with the responsibility of integrity and initiative. If you feel that something is important, then fight for it! We want a workplace where nobody feels like they're using the wrong tools for the job or doing the wrong thing. If they do, they're responsible for quickly voicing that concern, then proposing and executing upon solutions. Their teammates, then, are responsible for engaging in discussion around the topic and working together to make the best choice for everyone.
```

This doesn't mean that everything must constantly be in flux. Rather, it means that, when you believe something should happen, you either make it happen, or find a better solution.

> This also doesn't mean that you should make unilateral decisions. As an [open company](index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. There no shame in having your idea shot down or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one!

Reviewer 3: ```suggestion
This also doesn't mean that you should make unilateral decisions. As an [open company](index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. It is not a bad outcome if your proposal is not accepted or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one!
```

> This also doesn't mean that you should make unilateral decisions. As an [open company](index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. There no shame in having your idea shot down or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one!

Reviewer 2: ```suggestion
This also doesn't mean that you should make unilateral decisions. As an [open company](index.md#open-company), we value open discussion on important topics, and think that collaboration should be a large part of making things better. There is no shame in having your idea rejected or replaced with a better solution, because you took the initiative, started the conversation, and either reaffirmed that our current solution is good, or implemented an even better one!
```

Author: Took Nick's reworking of this block, but it was pretty similar
